site/app/controllers/NotificationSettings.php

```diff
@@ -13,8 +13,8 @@
  */
 class NotificationSettings extends AbstractController {
 
-	const NOTIFICATION_SELECTIONS = ['merge_threads', 'all_new_threads', 'all_new_posts', 'all_modifications_forum', 'reply_in_post_thread'];
-	const NUM = 5;
+	const NOTIFICATION_SELECTIONS = ['merge_threads', 'all_new_threads', 'all_new_posts', 'all_modifications_forum', 'reply_in_post_thread','merge_threads_email', 'all_new_threads_email', 'all_new_posts_email', 'all_modifications_forum_email', 'reply_in_post_thread_email'];
+	const NUM = 10;
```

Comment:
What is the point of NUM here? Is it the number of elements in NOTIFICATION_SETTINGS? If so, get rid of it and just do `count(NotificationSettings::NOTIFICATION_SELECTIONS)` wherever NUM was used.